core/src/test/java/org/apache/iceberg/TestMetricsTruncation.java

```diff
@@ -17,45 +17,44 @@
  * under the License.
  */
 
-package org.apache.iceberg.parquet;
+package org.apache.iceberg;
 
+import java.nio.ByteBuffer;
+import java.util.Comparator;
 import org.apache.iceberg.expressions.Literal;
 import org.junit.Assert;
 import org.junit.Test;
-import java.io.IOException;
-import java.nio.ByteBuffer;
-import java.util.Comparator;
 
 import static org.apache.iceberg.util.BinaryUtil.truncateBinaryMax;
 import static org.apache.iceberg.util.BinaryUtil.truncateBinaryMin;
 import static org.apache.iceberg.util.UnicodeUtil.truncateStringMax;
 import static org.apache.iceberg.util.UnicodeUtil.truncateStringMin;
 
-public class TestParquetMetricsTruncation {
+public class TestMetricsTruncation {
   @Test
-  public void testTruncateBinaryMin() throws IOException {
+  public void testTruncateBinaryMin() {
     ByteBuffer test1 = ByteBuffer.wrap(new byte[] {1, 1, (byte) 0xFF, 2});
     // Output of test1 when truncated to 2 bytes
-    ByteBuffer test1_2_expected = ByteBuffer.wrap(new byte[] {1, 1});
+    ByteBuffer test12Expected = ByteBuffer.wrap(new byte[] {1, 1});
```

Comment:
I think the underscores make the test more readable. Keeping them would also remove a lot of non-functional changes from this PR, so I think it is a good idea to revert the name style changes.

Reply:
I agree, but I'd need to disable the `checkstyle` rule for this.

Reply:
Can you add `@SuppressWarnings("checkstyle:(rulename)")` to the class?

Reply:
Just added. Thanks!